moveit_simple/src/moveit_simple.cpp

```diff
@@ -74,63 +74,67 @@ Robot::Robot(const ros::NodeHandle & nh, const std::string &robot_description,
 
 
 
-bool Robot::addTrajPoint(const std::string & traj_name, const Eigen::Affine3d pose,
+void Robot::addTrajPoint(const std::string & traj_name, const Eigen::Affine3d pose,
                   const std::string & frame, double time,
                   const std::string & point_name)
 {
   std::lock_guard<std::recursive_mutex> guard(m_);
 
-  bool success = false;
+//  bool success = false;
   ROS_INFO_STREAM("Attempting to add " << point_name << " to " << traj_name << "relative to"
                   << frame << " at time " << time);
-  try
-  {
+//  try
+//  {
     Eigen::Affine3d pose_rel_robot = transformToBase(pose, frame);
     std::unique_ptr<TrajectoryPoint> point =
       std::unique_ptr<TrajectoryPoint>(new CartTrajectoryPoint(pose_rel_robot, time, point_name));
-    success = addTrajPoint(traj_name, point);
-  }
-  catch (tf2::TransformException &ex)
-  {
-    ROS_WARN_STREAM("Add to trajectory failed for arbitrary pose point: " << ex.what());
-    success = false;
-  }
-  return success;
+    addTrajPoint(traj_name, point);
+
+//    success = addTrajPoint(traj_name, point);
+//  }
+//  catch (tf2::TransformException &ex)
+//  {
+//    ROS_WARN_STREAM("Add to trajectory failed for arbitrary pose point: " << ex.what());
+//    success = false;
+//  }
+//  return success;
 
 }
 
 
 
 
-bool Robot::addTrajPoint(const std::string & traj_name, const std::string & point_name,
+void Robot::addTrajPoint(const std::string & traj_name, const std::string & point_name,
                          double time)
 {
   std::lock_guard<std::recursive_mutex> guard(m_);
 
   ROS_INFO_STREAM("Attempting to add " << point_name << " to " << traj_name
                  << " at time " << time);
   std::unique_ptr<TrajectoryPoint> point = lookupTrajectoryPoint(point_name, time);
-  return addTrajPoint(traj_name, point);
+  addTrajPoint(traj_name, point);
 }
 
 
 
 
-bool Robot::addTrajPoint(const std::string & traj_name,
+void Robot::addTrajPoint(const std::string & traj_name,
                          std::unique_ptr<TrajectoryPoint> & point)
 {
   bool success = false;
   if( point )
   {
     traj_map_[traj_name].push_back(std::move(point));
-    success = true;
+//    success = true;
   }
   else
   {
+
     ROS_ERROR_STREAM("Failed to add point for trajectory " << traj_name);
-    success = false;
+    throw std::runtime_error("not a valid point to add");
```

Comment:
You should include the arguments to the function to make it easier for the developer to track down.  Say something like "Null point cannot be added to <traj_name>"